modAionImpl/src/org/aion/zero/impl/config/Network.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.zero.impl.config;
+
+import java.util.Arrays;
+
+/**
+ * Defines different Aion networks.
+ *
+ * @author Alexandra Roatis
+ */
+public enum Network {
+    MAINNET("mainnet", 256),
+    CONQUEST("conquest", 128),
+    MASTERY("mastery", 32),
+    CUSTOM("custom", 0);
+
+    private final String name;
+    private int identifier;
+
+    /**
+     * Constructor.
+     *
+     * @param _name network name
+     * @param _identifier network identifier
+     */
+    Network(String _name, int _identifier) {
+        this.name = _name;
+        this.identifier = _identifier;
+    }
+
+    @Override
+    public String toString() {
+        return this.name;
+    }
+
+    public int getIdentifier() {
+        return identifier;
+    }
+
+    /**
+     * Utility method that determines the correct network based on the given name.
+     *
+     * @param network a string value representing the network name
+     * @return the network object corresponding to the string value or null when the value is not
+     *     mapped to an object.
+     * @implNote There may be several test networks active. The term {@code testnet} is defined to
```

Comment:
would it be more convenient to just call it mastery instead of testnet then?

I'm just thinking in the future, if there's another testnet that becomes the "main" testnet, then all the old docs/scripts that refer to mastery as testnet will become confusing

Reply:
I disagree here. The alternative is to disallow the keyword which would end up with someone calling for `testnet` and getting the default `mainnet` without noticing the change. 

Additionally, an advantage to having this kind of default set is exactly for when a test network is being phased out. A user can still run on `mastery` by giving the keyword, but the default for new users will be the new network, enforcing the transition is a way. Updating documentation is simply a price to be paid when transitioning to a new test network.